### ISSUE TITLE
Lag with .5 to 2mb buffer, no video with higher buffer

### ISSUE THREAD

**Issue by minus1kelvin** [brycelarge/xteve-vpn#8]
When I set the buffer to 8mb Plex will churn and xteve says its playing but after a few seconds, Plex says it can't get the stream and then xteve just says the connection closed, no errors. If I decrease buffer to 0.5mb the stream will play but it stops and starts constantly. Any way I can increase my buffer to 8mb in the docker? I used 8mb in non-docker xteve so I assume its just some configuration adjustment I need to make.

**Comment by brycelarge** [comment]
I gave up long ago trying to use plex to watch IPTV. Emby works much better but these days I use GSE smart IPTV and an android app that my IPTV provider provides for their service.

Emby works better than plex for IPTV because it does not transcodes or remux the feed, well it’s an option at least.

Also I switched Plex dockers once which improved things but still I stopped using Plex but I still use xteve.

Kind Regards 
Bryce
